CONTRIBUTING.md

```diff
@@ -1,26 +1,6 @@
-# Contributing
-
-Information about contributing to Audacity is available at https://wiki.audacityteam.org/wiki/Contribute
-
-Visit that page to find up to date links.
-
-## Feedback
-
-* The Audacity forum is the best place to get help with using Audacity.
-* Suggestions for improvements to Audacity are best sent to the Audacity forum.
-* Bug reports to our feedback email address.
-
-## Developing
-
-* To discuss code for a proposed change, best to subscribe to audacity devel email list.
-* Code must be GPL v2 compatible.  
-
-## Translation
-
-* Best to subscribe to the translation email address
-
-## Supporting Users
-
-* Best done on the Audacity forum.
+# Contribution guidelines
+This project requires contributors to license their contributions under the GNU General Public License v2.0 or later (`GPL-2.0-or-later`). This allows it to use upstream code from Audacity when [they switch to GNU General Public License version 3](https://github.com/audacity/audacity/discussions/932).
 
+By contributing to this project, you hereby reaffirm that you are licensing your contribution under the GNU General Public License v2.0 or later (`GPL-2.0-or-later`).
```

Comment:
This line is pretty much a duplicate from the above. Can you remove it?

Reply:
Done